db/seeds.rb

```diff
@@ -47,3 +49,79 @@ class SeedAdminENVError < StandardError
 business_categories.each { |b_category| BusinessCategory.find_or_create_by(name: b_category) }
 BusinessCategory.find_or_create_by(name: 'Sociala tjänstehundar', description: 'Terapi-, vård- & skolhund dvs hundar som jobbar tillsammans med sin förare/ägare inom vård, skola och omsorg.')
 BusinessCategory.find_or_create_by(name: 'Civila tjänstehundar', description: 'Assistanshundar dvs hundar som jobbar åt sin ägare som service-, signal, diabetes, PH-hund mm')
+
+if Rails.env.development? || Rails.env.staging?
+  r = Random.new
+  USERS = 40
```

Comment:
minor note: my style would be to name this "NUM_USERS" to be more explicit about the intent.